_posts/en/newsletters/2020-10-07-newsletter.md

```diff
@@ -0,0 +1,198 @@
+    that reference specific UTXOs, allowing users to (arguably) prove
+    control over those funds, similar to [BIP127][] proof of reserves.
+
+    Alm is seeking feedback on the alternative proposal, including
+    whether or not it should replace BIP322 or be opened as a separate
+    BIP.
+
+- **Brief discussion about merging taproot in Bitcoin Core:** this
+  week's Bitcoin Core Developer Meeting included a [brief discussion][meeting
+  taproot] about whether the main taproot implementation ([PR
+  #19953][bitcoin core #19953]) should be merged in the next couple
+  weeks before the version 0.21 code is scheduled to be branched off,
+  whether it should be merged into the development branch shortly after
+  the branch off, or whether there should be no special ordering.  All
+  participants agreed any merge plans would be conditional on it
+  receiving sufficient review and no problems being found.
+
+    It seemed most participants favored merging it before the 0.21
+    branch off.  The taproot code in the PR does not contain any
+    activation parameters, so it will not be functional.  It'll be
+    similar to the way segwit code was included in [version 0.13.0][0.13
+    segwit prep] and then activation parameters were added in [version
+    0.13.1][].  To make this happen, taproot will need to finish being
+    reviewed before the project's [scheduled][bitcoin core #18947] 0.21
+    feature freeze on October 15th.  Several meeting participants said
+    they would make it their priority.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta][lnd 0.11.1-beta] is the release for a new minor
+  version.  Its release notes summarize the changes as "a number of
+  reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor
+  commitments][topic anchor outputs] spec compliant."
+
+- [HWI 1.2.0-rc.1][HWI 1.2.0] is a release candidate that adds support
+  for a new hardware device and contains multiple bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19898][] log: print unexpected version warning in validation log category FIXME:jonatack
+
+- [Bitcoin Core #15367][] feature: Added ability for users to add a startup command FIXME:dongcarl
+
+- [Bitcoin Core #19723][] ignores unknown P2P protocol messages sent
+  before the `verack` (version acknowledgment) message.  This allows a
```

Comment:
maybe something like "message, rather than considering them as peer misbehavior" to make the change from previous behavior more clear (like in line 125 below)

Reply:
Yes, I think something like this is needed. The change in n19723 allows future nodes to use the pre-verack feature negotiation method in a backwards-compatible way (i.e. that future node can send feature negotiation messages without being disconnected from old peers).